.circleci/config.yml

```diff
@@ -286,6 +286,12 @@ jobs:
       <<: *ci-template-vars
       EXTRA_PACKAGES: "time python autoconf automake"
 
+  elpi:
+    <<: *ci-template
+    environment:
+      <<: *ci-template-vars
+      EXTRA_OPAM: "ppx_tools_versioned ppx_deriving ocaml-migrate-parsetree"
```

Comment:
All opam packages should be installed in the opam-boot job because that's the one which generates the cache for future runs. Other jobs do not run opam.